### ISSUE TITLE
cant set HTTP_PROXY in `airflow.extraPipPackages` init-container

### ISSUE THREAD

**Issue by SingularBunny** [airflow-helm/charts#322]
You have incorrect helper [`airflow.init_container.install_pip_packages`](https://github.com/airflow-helm/charts/blob/42fdd8ceece5159e61e00cce94bb5812f0f8513a/charts/airflow/templates/_helpers/pods.tpl).
When it initializes, it doesn't contain `airflow-config` secret. 
Because of that many pods stuck if you use a proxy.

You need to add:
```
  envFrom:
    {{- include "airflow.envFrom" . | indent 4 }}
  env:
    {{- include "airflow.env" . | indent 4 }}
```
inside.

I am using version 8.4.1 of chart

Kubernetes Version: v1.21.3
Helm version: v3.6.3

Values:

airflow:

  extraEnv:
    - name: AIRFLOW__CORE__FERNET_KEY
      valueFrom:
        secretKeyRef:
          name: airflow
          key: fernet-key

  config:
    http_proxy: "http://host:8080"
    https_proxy: "http://host:8080"
    no_proxy: "localhost,kubernetes.default.svc,apiserver-loopback-client,127.0.0.0/8,10.0.0.0/8,10.12.50.77,postgresql.postgresql.svc.cluster.local,redis-master.redis.svc.cluster.local"
    HTTP_PROXY: "http://host:8080"
    HTTPS_PROXY: "http://host:8080"
    NO_PROXY: "localhost,kubernetes.default.svc,apiserver-loopback-client,127.0.0.0/8,10.0.0.0/8,10.12.50.77,postgresql.postgresql.svc.cluster.local,redis-master.redis.svc.cluster.local"

  extraPipPackages:
    - "airflow-exporter"
    - "apache-airflow[kubernetes]"

web:
  extraPipPackages:
    - "airflow-exporter"
    - "apache-airflow[kubernetes]"

logs:
  persistence:
    enabled: true
    existingClaim: airflow-logs-claim

dags:
  initContainer:
    enabled: false
    installRequirements: true

  persistence:
    enabled: true
    existingClaim: airflow-dags-claim

postgresql:
  enabled: false

externalDatabase:
  host: postgresql.postgresql.svc.cluster.local
  database: airflow_production
  user: airflow
  passwordSecret: postgresql-password

redis:
  enabled: false

externalRedis:
  host: redis-master.redis.svc.cluster.local
  passwordSecret: redis-password

**Comment by thesuperzapper** [comment]
@SingularBunny thanks for raising this, I have created https://github.com/airflow-helm/charts/issues/324 which describes what I plan to do.

**Comment by thesuperzapper** [comment]
@SingularBunny this feature will be available in version `8.5.0` of the chart (to be released today/tomorrow) because of PR: https://github.com/airflow-helm/charts/pull/365

__You can now set the `HTTP_PROXY` configs in `airflow.config`, for example:__
```yaml
airflow:
  config:
    HTTP_PROXY: http://example.com:8080
  
  extraPipPackages:
    - "xxxxxx==1.0.0"
```
